windowsforms/FlexGridShowcaseDemo/cs/FlexGridShowcaseDemo.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
AFAIK we don't add sln files, please remove

Reply:
Fixed with 2c8a711160f20afccabc2ee24021558164a47bab.